Project/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAgent.cs

```diff
@@ -37,6 +57,14 @@ public class WalkerAgent : Agent
 
     public override void Initialize()
     {
+        //Spawn an orientation cube
+        Vector3 oCubePos = hips.position;
```

Comment:
Typo ? Why    __o__ CubePos?

Reply:
Is it for orientationCubePos ? 
I think it should be spelled out since it is spelled out everywhere else.

Reply:
removed. refactored to include the cube as part of the prefab